src/async-wrap.cc

```diff
@@ -76,6 +76,8 @@ static void Initialize(Handle<Object> target,
 Handle<Value> AsyncWrap::MakeCallback(const Handle<Function> cb,
                                       int argc,
                                       Handle<Value>* argv) {
+  if (!env()->CanCallIntoJs())
+    return Undefined(env()->isolate());
```

Comment:
Is this comment supposed to be on some other line? The check here is to see if one can call into js not if `worker_context_ == nullptr`.